examples/nbody/nbody.cpp

```diff
@@ -211,6 +211,10 @@ struct UpdateKernel
             threads
         >;
 
+        auto threadBlockIndex  = alpaka::idx::getIdx<
```

Comment:
there is a space too much before `=`

Reply:
Seriously ;-)

Reply:
Yes the space was jumping directly in my face and it takes me 5min to get the additional space out of my head^^

Reply:
Than better do not look into this file at all before you apply my da3672f.

Reply:
ohh most of this was fine and following our [coding guide lines](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md) but I see we have not mentioned it somewhere.

Should I update your PR that you must not waste time into this. The problem is that no linter is supporting templates and we can therefore not do it automatically :-(

Reply:
I more or less choose /one/ arbitrary style used in this file and applied it.

Except for `&`, `&&`, and `*` I think I pretty much nailed it, didn't I?

Reply:
I pushed a second try, fell free to make your fixes on top of the 9b2473e7474ac4ab5b229e42c60b070611ac0ee0, so that I can better see, where I still was wrong. Thanks.

Reply:
Thanks for your update. I fixed only 3 or 4 small spaces issues. Did you use a tool for the reformatting or did you do it by hand? If you use a tool: which?

Reply:
ohh wait could you please mark on the right side the checkbox `Allow edits from maintainers.` else I can not push to your PR.

Reply:
no tool! but please rebase if you push!